source/client/process_impl.cc

```diff
@@ -848,8 +848,14 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const UriPtr& tracing_
     // Needs to happen as early as possible (before createWorkers()) in the instantiation to preempt
     // the objects that require stats.
     if (!options_.statsSinks().empty()) {
-      store_root_.setTagProducer(
-          Envoy::Config::StatsUtility::createTagProducer(bootstrap_, envoy_options_.statsTags()));
+      auto producer_or_error = Envoy::Stats::TagProducerImpl::createTagProducer(
```

Comment:
Would you mind listing the full type here instead of the `auto`? The type isn't obvious to me from the context.

Reply:
Done

Reply:
Thank you.